Gemfile

```diff
@@ -33,6 +33,7 @@ gem 'newrelic_rpm'
 gem 'pg'
 gem 'phonelib'
 gem 'pkcs11'
+gem 'prawn'
```

Comment:
What is the reason for adding this to the IdP instead of having a standalone thing? This seems like an easy place to avoid piling more onto the IdP.